packages/cascara/src/ui/Dashboard/Dashboard.js

```diff
@@ -81,7 +81,7 @@ const Dashboard = ({ as = 'div', className, config, ...props }) => {
 
   return (
     <Boundaries>
-      <Role {...props} as={as} className={cx(className, 'Dashboard')}>
+      <Role {...props} as={as} className={cx('Dashboard', className)}>
```

Comment:
```suggestion
      <Role {...rest} as={as} className={cx('Dashboard', rest.className)}>
```
I know that `className` is already being documented as part of propTypes, but we should remove that from existing code and make this be consistent. We can also change the `...props` in the destructure of the component to `...rest` for consistency.

Reply:
Done